src/aat/resources/features/F-026/F-026.feature

```diff
@@ -0,0 +1,58 @@
+@F-026
+Feature: F-026: Get a case data with UI layout
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-078
+  Scenario: must return the list of cases and status code 200 for correct inputs
+    Given a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [uses case-reference of the case just created]
```

Comment:
Why use the case reference of the case just created? And where is it actually used at all? No where...
This is promise to be met by the respective test data, which it indeed lists among spec, but there is no use of such case reference. Indeed, there shouldn't be, either.

Reply:
Updated: this is not necessary. removed.